src/pages/ExamsPage/index.jsx

```diff
@@ -25,19 +32,29 @@ const ExamsPage = ({ courseId }) => {
   const {
     attemptsList,
   } = useExamAttemptsData();
+  const {
+    allowancesList,
+  } = useAllowancesData();
 
   return (
     <Container>
       <Container id="exam-selector">
-        <ExamSelection exams={examsList} onSelect={setCurrentExam} />
+        <ExamSelection exams={examsList} onSelect={setCurrentExam} isDisabled={key === 'allowances'} />
       </Container>
-      <Tabs variant="tabs" mountOnEnter defaultActiveKey="attempts">
```

Comment:
Question: is there a reason that the Tabs component no longer needs the defaultActiveKey?